app/api-v1/routes/order.js

```diff
@@ -1,11 +1,16 @@
 // eslint-disable-next-line no-unused-vars
+const { BadRequestError } = require('../../utils/errors')
 module.exports = function (orderService) {
   const doc = {
     GET: async function (req, res) {
       res.status(500).json({ message: 'Not Implemented' })
     },
     POST: async function (req, res) {
-      res.status(500).json({ message: 'Not Implemented' })
+      const { statusCode, result } = await orderService.postOrder(req.body)
```

Comment:
we need to do some validation on the items list. They should all be valid `recipe` Ids within the database with the `supplier` equal to the `manufacturer` in the order